.editorconfig

```diff
@@ -7,82 +7,88 @@ root = true
 
 charset = utf-8
 
-#Core editorconfig formatting - indentation
+# Core editorconfig formatting - indentation
 
-#use hard tabs for indentation
+# use hard tabs for indentation
 indent_style = tab
 
-#Formatting - new line options
+# Formatting - new line options
 
 #require braces to be on a new line for all
 csharp_new_line_before_open_brace = all
 
-#Formatting - organize using options
+# Formatting - organize using options
 
-#do not place System.* using directives before other using directives
+# do not place System.* using directives before other using directives
 dotnet_sort_system_directives_first = true:error
 
-#Formatting - spacing options
+# Formatting - remove any whitespace characters preceding newline characters
+trim_trailing_whitespace = true
 
-#require NO space between a cast and the value
+# Formatting - set prefered newline characters
+end_of_line = lf
```

Comment:
Will this not make any issues?

Reply:
There's no simple answer. I don't know what platforms, code editors, etc. people use for developing Wasabi. However, the end result is appealing (at least to me).

Less aggressive approach would be to ask team members to modify that property locally and let them do some work and see whether they experience some issues or not.

Feel free to ACK / NACK this PR.